### PR TITLE
Sw 19100/new section labs

### PR DESCRIPTION
**Pull request by klarstil**

### CODE REVIEW

File: source/labs/api/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+shopware_version: X
+indexed: true
+group: Labs
+menu_title: API
+menu_order: 400
+---
+
+One of the most fundamental research projects we are currently working on is the new shopware API. With a completely 
+reworked and optimized architecture, the new API will be the foundation of both the front- and backend. Instead of being
+an extension to the existing infrastructure, the API will be regarded as a first-class citizen in shopware. We strive to
```
Comment (mitelg) [inline source/labs/api/index.md]: Shopware in uppercase as we are speaking of the software

File: source/labs/api/index.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+title: API
+github_link: labs/api/index.md
+shopware_version: X
+indexed: true
+group: Labs
+menu_title: API
+menu_order: 400
+---
+
+One of the most fundamental research projects we are currently working on is the new shopware API. With a completely
```
Comment (mitelg) [inline source/labs/api/index.md]: Shopware in uppercase

File: source/labs/housekeeping/filesystem-abstraction/index.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+To save your files on AWS S3, you have to modify your `config.php` and overwrite the filesystem you want to replace.
+
+The following example will store all `public` files on AWS S3:
+
+    'filesystem' => [
+        'public' => [
+            'type' => 'amazon-s3',
+            'config' => [
+                'bucket' => 'your-s3-bucket-name',
+                'region' => 'your-bucket-reg
```
Comment (mitelg) [inline source/labs/housekeeping/filesystem-abstraction/index.md]: did you missed something here?

File: source/labs/new-backend/index.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+title: New backend
+github_link: labs/new-backend/index.md
+shopware_version: X
+indexed: true
+group: Labs
+menu_title: New backend
+menu_order: 200
+---
+
+The new Shopware backend for the next Shopware version is another very interesting topic we're researching right now. In this article we would like to cover some of the topics we're working on.
```
Comment (mitelg) [inline source/labs/new-backend/index.md]: maybe one Shopware too much :-)

Comment (klarstil) [inline source/labs/new-backend/index.md]: 👍

File: source/labs/api/index.md (hunk 1 above)
Comment (klarstil) [inline source/labs/api/index.md]: 👍

File: source/labs/api/index.md (hunk 2 above)
Comment (klarstil) [inline source/labs/api/index.md]: 👍

File: source/labs/api/index.md (hunk 1 above)
Comment (klarstil) [inline source/labs/api/index.md]: :+1:

File: source/labs/housekeeping/filesystem-abstraction/index.md (hunk 3 above)
Comment (klarstil) [inline source/labs/housekeeping/filesystem-abstraction/index.md]: 👍  Good catch, I copied it straight from a internal document but I fixed it now

**Comment by klarstil** [comment]
@mitelg I applied the changes you requested and reworked the footer a little bit for a better usability on mobile.

**Pull request merged**